NOnion/Directory/TorDirectory.fs

```diff
@@ -252,7 +252,7 @@ type TorDirectory =
                             circuit.Create CircuitNodeDetail.FastCreate
                             |> Async.Ignore
 
-                        let consensusStream = TorStream circuit
+                        use consensusStream = new TorStream(circuit)
```

Comment:
@aarani are you sure parens are needed in this line now?

Reply:
AFAIK parens are needed when "new" comes

Reply:
I checked again, I was right

Reply:
I'll mark this as resolved.